schemas/com.snowplowanalytics.snowplow.enrichments/pii_enrichment_config/jsonschema/2-0-0

```diff
@@ -0,0 +1,195 @@
+                      "se_category",
+                      "se_action",
+                      "se_label",
+                      "se_property",
+                      "mkt_clickid",
+                      "refr_domain_userid",
+                      "domain_sessionid"
+                    ]
+                  }
+                },
+                "required": [
+                  "field"
+                ],
+                "additionalProperties": false
+              },
+              "json": {
+                "description": "JSON field which contains a JSON string value, on which pseudonymization will be performed on a specific JSON path",
+                "type": "object",
+                "properties": {
+                  "field": {
+                    "enum": [
+                      "contexts",
+                      "derived_contexts",
+                      "unstruct_event"
+                    ]
+                  },
+                  "schemaCriterion": {
+                    "type": "string",
+                    "pattern": "^iglu:([a-zA-Z0-9-_.]+)/([a-zA-Z0-9-_]+)/([a-zA-Z0-9-_]+)/([1-9][0-9]*|\\*)-((?:0|[1-9][0-9]*)|\\*)-((?:0|[1-9][0-9]*)|\\*)$"
+                  },
+                  "jsonPath": {
+                    "type": "string",
+                    "pattern": "^\\$.*$"
+                  }
+                },
+                "required": [
+                  "field",
+                  "schemaCriterion",
+                  "jsonPath"
+                ],
+                "additionalProperties": false
+              }
+            },
+            "oneOf": [
+              {
+                "required": [
+                  "pojo"
+                ]
+              },
+              {
+                "required": [
+                  "json"
+                ]
+              }
+            ],
+            "additionalProperties": false
+          }
+        },
+        "strategy": {
+          "description": "The pseudonymization strategy which will be applied to all the fields specified in the `pii` section",
+          "type": "object",
+          "properties": {
+            "pseudonymize": {
+              "description": "Pseudonymization strategy that hashes using a specified algorithm",
+              "type": "object",
+              "properties": {
+                "hashFunction": {
+                  "description": "The hash function that will be used by this strategy",
+                  "enum": [
+                    "MD2",
+                    "MD5",
+                    "SHA-1",
+                    "SHA-256",
+                    "SHA-384",
+                    "SHA-512"
+                  ]
+                },
+                "salt": {
+                  "description": "Describes the method of obtaining a salt that will be added to the field durring hashing",
+                  "type": "object",
+                  "properties": {
+                    "ec2ParameterStore": {
```

Comment:
Can we change it to exact [same format](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/2-1-0#L36-L50) as it is in storage configuration? I mean `[string, object]` insetad of `oneOf: [plainValue, ec2ParameterStore]` I probably like your approach little bit more, but just in sake of consistency.

Reply:
Yes that's probably a good idea.

Reply:
@chuwy Can you please check the last commit, to see if that is what you meant?

Reply:
No, sorry, not exactly. `plainValue` is redundant. You just specify `"salt": "foobar"` when you need plain value, not `"salt": { "plainValue": "foobar" }

Reply:
Done

Reply:
Done. I have to say, I don't think this is an improvement over the original

Reply:
> I don't think this is an improvement over the original

I agree with that. But that was just in sake of consistency.